scripts/asr/transcribe_file_offline.py

```diff
@@ -39,14 +39,18 @@ def main() -> None:
     riva.client.add_word_boosting_to_config(config, args.boosted_lm_words, args.boosted_lm_score)
     riva.client.add_speaker_diarization_to_config(config, args.speaker_diarization)
     riva.client.add_endpoint_parameters_to_config(
-        config,
-        args.start_history,
-        args.start_threshold,
-        args.stop_history,
-        args.stop_history_eou,
+        config, 
+        args.start_history, 
+        args.start_threshold, 
+        args.stop_history, 
+        args.stop_history_eou, 
```

Comment:
trailing whitespaces can be stripped

Reply:
We should configure pre-commit for clients.

Reply:
Updated